project/AppDependencies.scala

```diff
@@ -3,12 +3,12 @@ import sbt.*
 object AppDependencies {
 
   private val hmrcMongoPlayVersion = "2.0.0"
-  private val bootstrapVersion     = "8.5.0"
+  private val bootstrapVersion     = "8.6.0"
 
   val compile: Seq[ModuleID] = Seq(
     "uk.gov.hmrc"                  %% "bootstrap-frontend-play-30" % bootstrapVersion,
     "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-30"         % hmrcMongoPlayVersion,
-    "uk.gov.hmrc"                  %% "play-frontend-hmrc-play-30" % "8.5.0",
+    "uk.gov.hmrc"                  %% "play-frontend-hmrc-play-30" % "9.11.0",
```

Comment:
Upgrade to 10.1.0 - There shouldn't be an issue with this

Reply:
I thought we will use 10.x version for scala 3 jira and we will update all deps related to this, I can update not sure how safe it is